core/src/main/scala/org/apache/spark/ExecutorAllocationClient.scala

```diff
@@ -59,6 +59,19 @@ private[spark] trait ExecutorAllocationClient {
   def killExecutors(executorIds: Seq[String]): Seq[String]
 
   /**
+   * Request that the cluster manager try harder to kill the specified executors,
+   * and maybe replace them.
+   * @return whether the request is acknowledged by the cluster manager.
```

Comment:
I've copied the doc from CoarseGrainedSchedulerBackend here and changed the incorrect `return` doc.